stock_orderpoint_uom/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Orderpoint UoM",
+    "summary": "Allows to create procurement orders in the UoM indicated in "
+               "the orderpoint",
+    "version": "11.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://www.odoo-community.org",
```

Comment:
https://github.com/OCA/stock-logistics-warehouse

Reply:
Corresponded. Thank you.